job_definitions/notify_buyers_to_award_closed_briefs_4_weeks.yml

```diff
@@ -0,0 +1,73 @@
+{% set environments = ['production'] %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-buyers-to-award-closed-briefs-4-weeks-{{ environment }}"
+    display-name: "Notify buyers to award closed briefs (4 weeks) - {{ environment }}"
+    project-type: freestyle
+    description: "Send first email notification to buyer users to award closed briefs, after 4 weeks."
+    parameters:
+      - string:
+          name: BUYER_IDS
+          description: "List of buyer IDs to be emailed e.g. 3,4,6"
+      - string:
+          name: DATE_CLOSED
+          description: "Notify buyers for briefs on a specific closing date (YYYY-MM-DD) - must be earlier than the offset days (28 or 56)"
+      - string:
+          name: OFFSET_DAYS
+          default: 28
+          description: "Notify buyers for briefs that closed X days ago"
+      - string:
+          name: NOTIFY_TEMPLATE_ID
+          default: "204d2bf5-5dd3-44c8-a68f-9111e040e831"
+          description: "Notify template ID for 4-week email"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    scm:
+      - git:
+          url: git@github.com:alphagov/digitalmarketplace-scripts.git
+          credentials-id: github_com_and_enterprise
+          branches:
+            - master
+          wipe-workspace: false
+    triggers:
+      - timed: "H 8 * * *"
+    publishers:
+      - trigger-parameterized-builds:
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
+            predefined-parameters: |
+              USERNAME=award-closed-briefs-4-weeks
+              JOB=Notify buyers to award closed briefs after 4 weeks {{ environment }}
+              ICON=:question:
```

Comment:
This is the slack icon that appears when the job fails. We are already using the :question: for the question and answers jenkins job. Not that we need to have different icons for each... but this is your chance to unleash your creative side and pick an appropriate emoji...

Reply:
Probably not appropriate icons but I don't think they've been used by anyone else yet 😸